easycv/models/utils/dist_utils.py

```diff
@@ -4,9 +4,13 @@
 import torch
 from mmcv.runner import get_dist_info
 from pytorch_metric_learning.utils import common_functions as c_f
-from torch.distributed import ReduceOp
 from torch.nn.parallel import DistributedDataParallel as DDP
 
+try:
+    from torch.distributed import ReduceOp
+except ImportError:
+    raise ImportError('Blade does not support ReduceOp')
```

Comment:
If you use blade, it will definitely raise error here, the ReduceOp should be encapsulated within the calling api